packages/kit/src/views/Earn/components/Recommended.tsx

```
@@ -0,0 +1,326 @@
+        logoURI: token.logoURI,
+      });
+    }
+  }, [account?.id, indexedAccount?.id, toTokenProviderListPage, token]);
+
+  if (!token) {
+    return <YStack width="$40" flexGrow={1} />;
+  }
+
+  return (
+    <YStack
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)